Scripts/Services/LootGeneration/RunicReforging/RunicReforging.cs

```diff
@@ -1992,7 +1981,7 @@ public static void GenerateRandomItem(Item item, Mobile killer, int basebudget,
                         mods = Math.Max(minmods, GetProperties(maxmods));
 
                         perchigh = 100;
-                        perclow = Utility.RandomMinMax(50, 70);
+                        perclow = Convert.ToInt32(Utility.RandomMinMax(50, 70) * perc);
```

Comment:
Same as above, this 50-70 was always ignored. So its being reduced against the perc rolled at the start. We would see less Legendary artifacts with 1% properties and more higher values. But 1% is still possible on bad real life luck.